src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -136,6 +136,7 @@ public static IEdmTerm FindTerm(this IEdmModel model, string qualifiedName)
             EdmUtil.CheckArgumentNull(model, "model");
             EdmUtil.CheckArgumentNull(qualifiedName, "qualifiedName");
 
+            qualifiedName = model.ReplaceAlias(qualifiedName);

```

Comment:
I only noticed a couple of functional changes, but this one looks like it could use a unit test around it